Gemfile

```diff
@@ -0,0 +1,15 @@
+ruby '2.2.2'
+
+gem 'activesupport', '~> 4.2.2'
+gem 'elasticsearch-model'
+gem 'activemodel'
+gem 'typhoeus'
+gem 'activerecord'
```

Comment:
* I think I can move `ActiveRecord` to a dev dependency as you suggest and keep `ActiveModel` as I think I only need to reference the `after_save` and `after_delete` callbacks.
* The [elasticsearch-transport](https://github.com/elastic/elasticsearch-ruby/tree/master/elasticsearch-transport) gem, used under the hood, pulls in `Faraday` and detects the presence of Typhoeus automatically, all that it needs is a `require 'typhoeus'; require 'typhoeus/adapters/faraday'` in order to use Typhoeus for transport, so I would need to add that `require` in order to make this dependency useful. The question is, should we add support for Typhoeus here or leave it upto the client app? I'm probably in favour of the former but it does add an extra dependency.